pkg/vm/vm.go

```diff
@@ -59,6 +63,9 @@ type VM struct {
 	estack *Stack // execution stack.
 	astack *Stack // alt stack.
 
+	stackItemCount int  // total item count
```

Comment:
'.'

Reply:
I believe that should stay only in doc comments. All other comments I have encounted have no dots after (because it is not a sentence).